analytical_engine/java/grape-giraph/pom.xml

```diff
@@ -78,10 +78,10 @@
       <artifactId>giraph-core</artifactId>
       <version>1.3.0-hadoop2</version>
     </dependency>
-    <dependency>
+    <!-- <dependency>
       <groupId>org.scijava</groupId>
       <artifactId>native-lib-loader</artifactId>
-    </dependency>
+    </dependency> -->
```

Comment:
Remove if not needed. Just commenting it would leave a lot of confusion for other developers.

Reply:
Yep, will be removed.